chsdi/models/vector/uvek.py

```diff
@@ -148,37 +148,73 @@ class MountainbikelandSperrungenUmleitungen(Base, SperrungenUmleitungen, Vector)
 
 
 class OevHaltestellen:
+    __bodId__ = 'ch.bav.haltestellen-oev'
```

Comment:
It does not make much sence to me to create a class with only one element.

You could also add the following elements is I saw correctly:
- `id`
- `__table_args__` 

Or perhaps make a parent class for OevHaltestellenZoom1 and OevHaltestellenZoom2 with subclasses and an individual calss for OevHaltekante

Reply:
I thought about it but it was the best solution I found as the update was a bit tricky (I spent some time on it believe me :)) Beside I didn't find a way to attach the new haltekante class to the existing haltestellen bod layer without doing it (it wasn't working otherwise). I'm aware of the redundancy in the Zoom classes. I can try to optimize it now that the layer is accepted...